torchaudio/backend/sox_io_backend.py

```diff
@@ -184,6 +184,9 @@ def load_wav(
 ) -> Tuple[torch.Tensor, int]:
     """Load wave file.
 
+    Warnings:
```

Comment:
I think this is unnecessary, because it's duplicated with the warning statement and because we don't do this in any other library.